components/Blocks/PhotoHero.css

```diff
@@ -0,0 +1,46 @@
+.root {
+  width: 100%;
+  padding: 5em 0;
+  position: relative;
+  background-size: cover;
+  color: var(--color-white);
```

Comment:
Should we abstract this out so we use a non-colour variable name here? `--color-hero-text` or something. Just in case we ever change shit and we’re not overwriting our explicit colour variables.